.github/workflows/test.yml

```diff
@@ -1,16 +1,20 @@
 name: test
 
 on:
-  push:
-    branches: [main]
-  pull_request:
+  workflow_call:
 
 jobs:
   test:
     runs-on: ubuntu-latest-16-cores
```

Comment:
do we really need this expensive runner?

Reply:
Not for now, my mistake